File: 两数求和.cpp
class Solution {
public:
    vector<int> twoSum(vector<int>& nums, int target) {
        vector<int> res;
        map<int, int> hash;
        for (int i = 0; i < nums.size(); i++)
        {
            int temp = target - nums[i];
            if(hash.count(temp))
            {
                res = vector<int> ({hash[temp], i});
                return res;
            }
            hash[nums[i]] = i;
        }
        return res;
    }
};

File: 两数求和.py
"""
思路：定义dic，利用dic的enumerate方法，利用求得key去寻找value，然后输出两个
"""
def twoSum(self, nums, target):
        """
        :type nums: List[int]
        :type target: int
        :rtype: List[int]

        """
        dic = {}
        for index, item in enumerate(nums):
            if item in dic:
                return[dic[item], index]
            else:
                dic[target - item] = index
